### ISSUE TITLE
Error on https://ruffle.rs/demo/

### ISSUE THREAD

**Issue by Misc7** [ruffle-rs/ruffle#8670]
###  Error Info
Error name: Error
Error message: panicked at 'Stage must always have a root movie', core/src/display_object/stage.rs:580:9
Error stack:
```
Error: panicked at 'Stage must always have a root movie', core/src/display_object/stage.rs:580:9
    at n.wbg.__wbg_new_f9bb493369a83c12 (https://ruffle.rs/demo/966.index.js:1:5454)
    at std::sync::once::Once::call_once::{{closure}}::h4e9b55c483913b51 (wasm://wasm/01aafea2:wasm-function[1161]:0x2bdd5e)
    at std::sync::once::Once::call_inner::h34aee843864daceb (wasm://wasm/01aafea2:wasm-function[1495]:0x339ec3)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::heee5085b8677f23c (wasm://wasm/01aafea2:wasm-function[5311]:0x5005ab)
    at std::panicking::rust_panic_with_hook::h70a0e195f4db2a29 (wasm://wasm/01aafea2:wasm-function[3984]:0x4b3150)
    at core::panicking::panic_fmt::h751be80779d42b53 (wasm://wasm/01aafea2:wasm-function[5572]:0x50a39a)
    at core::option::expect_failed::h2290d18a0c9a6a24 (wasm://wasm/01aafea2:wasm-function[6326]:0x520317)
    at ruffle_core::display_object::stage::Stage::root_clip::h477f7bb016578214 (wasm://wasm/01aafea2:wasm-function[3922]:0x4ad42f)
    at ruffle_web::Ruffle::tick::hf0e11324b0afb2df (wasm://wasm/01aafea2:wasm-function[717]:0x201436)
    at ruffle_web::Ruffle::new_internal::{{closure}}::{{closure}}::{{closure}}::h8a0e13681ef1464c (wasm://wasm/01aafea2:wasm-function[6596]:0x52b3c3)
```

# Player Info
Allows script access: false
Renderer: WebGL

# Page Info
Page URL: https://ruffle.rs/demo/

# Browser Info
User Agent: Mozilla/5.0 (X11; CrOS x86_64 15117.112.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36
Platform: Linux x86_64
Has touch support: true

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-12-02
Channel: nightly
Built: 2022-12-02T00:19:08.183Z
Commit: 3554b041b4fb134fd4c7d8bfa8a4172e5212e5c5
Is extension: false

# Metadata
width: 640
height: 480
frameRate: 30
numFrames: 2
swfVersion: 10
backgroundColor: undefined
isActionScript3: true
uncompressedLength: 1381898

**Comment by n0samu** [comment]
Hi, what SWF file were you trying to play?

**Comment by n0samu** [comment]
This was probably fixed by #10026
